### PR TITLE
Release ppx_deriving_yojson.3.9.1

### PR DESCRIPTION
**Pull request by NathanReb**
**CHANGES**

  * Fix generation of unnecessary `Poly_typ ([], ...)` nodes when deriving de/serializer for open types. These are rejected by OCaml 5.3 onward. (#162) @NathanReb

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
for opam-repository maintainers or you if you have the time: once merged it might be worth restricting all previous releases of ppx_deriving_yojson to OCaml < 5.3 (see https://github.com/ocaml-ppx/ppx_deriving_yojson/issues/161) using
```
opam admin add-constraint 'ocaml<5.3' --packages ppx_deriving_yojson
```

File: packages/ppx_deriving_yojson/ppx_deriving_yojson.3.9.1/opam [hunk 1]
```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+version: "3.7.0"
```
[the remark this answers is not in the document]
Comment (NathanReb) [reply packages/ppx_deriving_yojson/ppx_deriving_yojson.3.9.1/opam]: Ouh my bad! Thanks!

**Comment by NathanReb** [comment]
> for opam-repository maintainers or you if you have the time: once merged it might be worth restricting all previous releases of ppx_deriving_yojson to OCaml < 5.3 (see ocaml-ppx/ppx_deriving_yojson#161) using

Sure thing, I'll do it!

**Comment by NathanReb** [comment]
I think the revdeps failure are unrelated, I'll proceed with the actual release!

**Comment by mseri** [comment]
Just observed the issue with previous derivers:
```
#=== ERROR while compiling ojs_base.0.8.0 =====================================#
# context              2.4.0~alpha1~dev | linux/x86_64 | ocaml-base-compiler.5.3.0~beta1 | pinned(https://zoggy.frama.io/ojs-base/releases/ojs-base-0.8.0.tar.bz2)
# path                 ~/.opam/5.3~beta1/.opam-switch/build/ojs_base.0.8.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p ojs_base -j 31 --promote-install-files=false @install
# exit-code            1
# env-file             ~/.opam/log/ojs_base-7-bc7d97.env
# output-file          ~/.opam/log/ojs_base-7-bc7d97.out
### output ###
# (cd _build/default && /home/opam/.opam/5.3~beta1/bin/ocamlc.opt -g -w -40 -bin-annot -w -6-7-9-10-27-32-33-34-35-36-39-50-52 -no-strict-sequence -g -bin-annot -bin-annot-occurrences -I lib/.ojs_base.objs/byte -I /home/opam/.opam/5.3~beta1/lib/lwt -I /home/opam/.opam/5.3~beta1/lib/ppx_deriving/runtime -I /home/opam/.opam/5.3~beta1/lib/ppx_deriving_yojson/runtime -I /home/opam/.opam/5.3~beta1/lib/seq -I /home/opam/.opam/5.3~beta1/lib/yojson -no-alias-deps -open Ojs_base -o lib/.ojs_base.objs/byte/ojs_base__Types.cmi -c -intf lib/types.pp.mli)
# File "_none_", line 1:
# Error: broken invariant in parsetree: Explicit universal type quantification cannot be empty.
```

**Comment by mseri** [comment]
The CI is re-running, so I need to wait for it to finish before I can merge

**Comment by NathanReb** [comment]
This issue is the one that this version fixes, or is supposed to fix at least. Where did you get that error?

**Comment by mseri** [comment]
Looks all right, thanks!

**Comment by mseri** [comment]
This was before this PR was merged indeed, seen on https://github.com/ocaml/opam-repository/pull/27081

**Comment by NathanReb** [comment]
Ah nice, thanks for letting me know!

**Pull request merged**